src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -3498,3 +3499,49 @@ class CopyStatementSegment(BaseSegment):
             ),
         ),
     )
+
+@postgres_dialect.segment()
+class LanguageClauseSegment(BaseSegment):
+    """A clause for specifying the language used for executing anonymous code
+    blocks.
+    """
+
+    type = "language_clause"
+
+    match_grammar = Sequence(
+        "LANGUAGE",
+        Ref("ParameterNameSegment")
+    )
+
+@postgres_dialect.segment()
+class DoStatementSegment(BaseSegment):
+    """A `DO` statement for executing anonymous code blocks.
+
+    As specified in https://www.postgresql.org/docs/14/sql-do.html
+    """
+
+    type = "do_statement"
+
+    _lang_first_clause = Sequence(
```

Comment:
Could you please move these (`_lang_first_clause` and `_lang_last_clause`) either inside the `Sequence()` below or define them as grammars instead? That's the standard style we've adopted across all the dialects. There's nothing "wrong" with it as written, this is just to avoid confusing people by having multiple ways of doing things.

Reply:
Done!